addons/l10n_ke/models/l10n_ke_item_code.py

```diff
@@ -0,0 +1,21 @@
+
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields, api
+
+
+class L10nKeItemCode(models.Model):
+    _name = 'l10n.ke.item.code'
+
+    name = fields.Char(compute='_compute_name', store=True)
```

Comment:
Could not the name be the code and the display_name be used to concatenate both?  There is even a way to add that in the class immediately.